homeassistant/components/homewizard_climate/climate.py

```diff
@@ -0,0 +1,226 @@
+"""Climate platform for homewizard_climate."""
+import logging
+import time
+
+from homewizard_climate_websocket.model.climate_device_state import (
+    HomeWizardClimateDeviceState,
+)
+from homewizard_climate_websocket.ws.hw_websocket import HomeWizardClimateWebSocket
+
+from homeassistant.components.climate import (
+    FAN_HIGH,
+    FAN_LOW,
+    FAN_MEDIUM,
+    FAN_OFF,
+    FAN_ON,
+    SWING_HORIZONTAL,
+    SWING_OFF,
+    ClimateEntity,
+    ClimateEntityFeature,
+    HVACMode,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import ATTR_TEMPERATURE, TEMP_CELSIUS
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Todo."""
+    websockets = hass.data[DOMAIN][entry.entry_id]["websockets"]
+    entities = [HomeWizardClimateEntity(ws, hass) for ws in websockets]
+    async_add_entities(entities)
+
+
+class HomeWizardClimateEntity(ClimateEntity):
+    """Todo."""
+
+    def __init__(
```

Comment:
It would be nice that `_attr_device_info` is filled in. This creates the device entry were automations can be created easily.

Reply:
aah, didn't know that thanks. Added in the form of a property.
I can change it to `self._attr_device_info`, according to the outcome of [this comment](https://github.com/home-assistant/core/pull/84105#discussion_r1052625272).